### ISSUE TITLE
Render object above another object while neglecting depth

### ISSUE THREAD

**Issue by Robin-Sb** [JirkaDellOro/FUDGE#217]
It should be possible to render an object above another object while neglecting the depth. 

Possible solutions could be:
1. overlay the main canvas with another canvas and draw the object there.
2. use one canvas but two viewports and draw the content of the second viewport afterwards.
3. disable the depth check before the other object is drawn. 

This list is not exhaustive; there might be other possible solutions.
This is supposed to be discussion/reminder thread with the intend to find the best solution.

**Comment by JirkaDellOro** [comment]
I think we added a method to render for en/disabling DepthTest, correct? Is this still relevant?

**Comment by Robin-Sb** [comment]
I don't think we did, we only added a method to enable/disable backface culling. 
I am also not sure how a method to toggle the depth test globally solves the issue.

As far as I am concerned, this is still relevant.
